code/modules/admin/verbs/adminhelp.dm

```diff
@@ -175,7 +175,11 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	id = ++ticket_counter
 	opened_at = world.time
 
-	name = msg
+	if(length(msg) < 100)
+		name = msg
+	else
+		var/breakmsg = findchar(msg, "#") || 100
+		name = copytext(msg, 1, breakmsg)

```

Comment:
length() returns the length in bytes, not characters. Use length_char().
findchar() returns a byte position but the 100 is a character count. You'll have to do a copytext_char(msg, 1, 100) first and then check for #. You need to do it no matter if there's a # or not or else someone could send a 10,000 character ahelp and it would pass wholly if the last char was #. Which is wrong.

Reply:
Oh right, we utf-8 now. Thanks

Reply:
Allright, skipped the linebreak thing so now it just cuts at 100. I did mess up that breakmsg, good catch heh